src/loaddb/load_worker_manager.cpp

```diff
@@ -166,6 +174,27 @@ namespace cubload
       }
 
     g_wp_mutex.unlock ();
+    g_wp_condvar.notify_one ();
+  }
+
+  void
+  worker_manager_stop_all ()
+  {
+    std::unique_lock<std::mutex> ulock (g_wp_mutex);
+    if (g_active_sessions.empty ())
+      {
+	return;
+      }
+
+    for (auto &it : g_active_sessions)
+      {
+	it->interrupt ();
```

Comment:
I think session needs to be deleted for this case

Reply:
won't load session be deleted by either sloaddb_destroy or session_state_uninit?

Reply:
in case we are sure that one of these functions is called on shutdown then it is safe

Reply:
I think it is safer than deleting here; I am not sure load client will expect a deleted session.